src/test/java/com/google/api/codegen/testdata/discoveries/foo.v1.json.overrides

```diff
@@ -4,5 +4,27 @@
   },
   "js|python": {
     "discoveryDocUrl": "localhost:8080/$discovery/foo?version=v1"
+  },
+  "csharp|go|java": {
+    "methods": {
+      "foo.bar.get": {
+        "fields": {
+          "foobar": {
+            "defaultValue": "\" \""
```

Comment:
I've included the override for foo as a demonstration, it's not actually necessary.

I need this field for my overrides generator for the mock tests, which automatically populates these fields with values. The reason is that C# requires a non-empty value in a repeated field.

No production sample should ever use this.